### ISSUE TITLE
Error while uninstalling rancher backups app

### ISSUE THREAD

**Issue by sowmyav27** [rancher/dashboard#2037]
On 2.5-head - commit id: `395b4fe77`

- Install rancher-backups: v1.0.300
- Select `rancher-backup` and `rancher-backup-crd` and click on "Delete" after they have been successfully installed.
- `rancher-backup-crd` gets deleted first
- Error in `rancher-backup` uninstallation logs: 

```
helm uninstall --namespace=cattle-resources-system rancher-backup
Error: uninstallation completed with 1 error(s): unable to build kubernetes objects for delete: unable to recognize "": no matches for kind "ResourceSet" in version "resources.cattle.io/v1"
```

**Comment by cbron** [comment]
The error happens because the CRD chart was removed first, that is not a valid path. You must always uninstall the primary chart before the CRD chart.

**Comment by sowmyav27** [comment]
@cbron I usually select both the app and the CRD and hit delete. (From the dashboard UI). Are you saying we should select the app first and delete and then the CRD. ?

**Comment by cbron** [comment]
@sowmyav27 yes that is solution for now. Maybe we can find a better solution in future.

**Comment by vincent99** [comment]
This is not really a UI task, it's just sending delete calls and has no idea which ones depend on which others or when it's safe to delete which and you can create the same situation with `helm remove`. If we want that there needs to be enough info to determine the dependency graph and the API should orchestrate walking it.
